app/src/main/java/org/mozilla/fenix/settings/logins/view/EditLoginView.kt

```diff
@@ -0,0 +1,58 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.settings.logins.view
+
+import android.text.InputType
+import android.view.ViewGroup
+import androidx.core.content.res.ResourcesCompat
+import kotlinx.android.extensions.LayoutContainer
+import kotlinx.android.synthetic.main.fragment_edit_login.view.*
+import org.mozilla.fenix.R
+import org.mozilla.fenix.components.metrics.Event
+import org.mozilla.fenix.ext.components
+
+/**
+ * View that contains and configures the Edit Login screen
+ */
+@Suppress("ForbiddenComment")
+class EditLoginView(
+    override val containerView: ViewGroup
+) : LayoutContainer {
+    private val context = containerView.context
+
+    // TODO: create helper class for toggling passwords. Used in login info and edit fragments.
```

Comment:
Let's file an issue for this if we haven't already and link to it in the comment